docs/developer/extending/apps/synchronous-webhooks/tax-webhooks.mdx

```diff
@@ -0,0 +1,357 @@
+
+:::note
+For amount fields, we use the same format as in the rest of the API.
+For example amount of 12,34 USD will be sent as `"total_amount": "12.34"`
+:::
+
+:::warning
+
+- All sales are included in line prices.
+- Shipping vouchers are included in shipping prices.
+- Vouchers for "Specific products" are included in line prices.
+- Vouchers "Apply only to a single cheapest eligible product" are included in line prices.
+  :::
+
+## Request format
+
+```json
+[
+  {
+    "type": "Order",
+    "id": "T3JkZXI6ZTdlODYzMDctMDdhYi00ODNjLTk0YmUtM2FiNzlkYTA2ZGRm",
+    "included_taxes_in_prices": true,
+    "shipping_name": "FedEx",
+    "shipping_amount": "59.17",
+    "currency": "USD"
+    "metadata": {},
+    "user_id": "VXNlcjo3OTU=",
+    "user_public_metadata": {},
+    "address": {
+      "type": "Address",
+      "id": "QWRkcmVzczoyMDM=",
+      "first_name": "Brandon",
+      "last_name": "Ruiz",
+      "company_name": "",
+      "street_address_1": "09925 Isabel Run",
+      "street_address_2": "",
+      "city": "JAMESFORT",
+      "city_area": "",
+      "postal_code": "72850",
+      "country": "US",
+      "country_area": "AR",
+      "phone": ""
+    },
+    "channel": {
+      "type": "Channel",
+      "id": "Q2hhbm5lbDox",
+      "slug": "default-channel",
+      "currency_code": "USD"
+    },
+    "discounts": [
+      {
+        "name": "New user discount",
+        "amount": "10.00"
+      }
+    ],
+    "lines": [
+      {
+        "type": "OrderLine",
+        "id": "T3JkZXJMaW5lOmE5MTc2NjhmLWNmYjUtNDM4Yi05MTFlLWNhNGVlZTgyYzQ5Mw==",
+        "variant_id": "UHJvZHVjdFZhcmlhbnQ6Mzg0",
+        "full_name": "Apple Juice (202165300)",
+        "product_metadata": {},
+        "product_type_metadata": {},
+        "charge_taxes": true,
+        "sku": "202165300",
+        "unit_amount": "1.99",
+        "total_amount": "1.99",
+        "product_name": "Apple Juice",
+        "variant_name": "202165300",
+        "quantity": 1
+      }
+    ],
+  }
+]
+```
+
+### Fields:
+
+1. `id` - Id of order for which app should calculate taxes.
+2. `included_taxes_in_prices` - Flag that shows whether taxes should be included in prices.
+3. `shipping_name` - Name of shipping method assigned to order.
+4. `shipping_amount` - Price of shipping method assigned to order.
+5. `currency` - Order currency.
+6. `metadata` - Public metadata of order.
+7. `user_id` - Id of the user assigned to order. (In case of anonymous customer this field return `null`)
+8. `user_public_metadata` - Public metadata of user assigned to order.
+9. `address` - Shipping address assigned to order. (In case of order without required shipping this field contains billing address.)
```

Comment:
```suggestion
9. `address` - Shipping address assigned to the order. (In case of the order without required shipping this field contains billing address.)
```